### DISCUSSION TITLE
¿Cómo continuar a la misión backend?

### DISCUSSION THREAD

**Discussion by FernandaOchoa** [LaunchX-InnovaccionVirtual/CursoIntroPython#345] [Announcements]
![](https://fernandaochoa.github.io/innovatemplate/images/posts/banner.png)

## 👋 Hola  @LaunchX-InnovaccionVirtual/mission-innova001 !

Hace unos días tuvimos un  live en  [el instagram de Innovacción Virtual](https://www.instagram.com/innovaccionvirtual/) 

Con algunas noticias interesantes, si te lo perdiste, puedes revisar todos los detalles y los siguientes pasos para continuar a la misón backend, incluso si no estuviste en la misión frontend. Sí, leíste bien, puedes avanzar directamente a la misión backend aún cuando no hayas estado en la misión frontend.
  
  ¿Apoco se podía? Inicialmente no. Sin embargo, sabemos que no a todes nos encanta ser fullstack por lo que en la vida real puedes ser sólamente frontend o sólamente backend. Tendremos un live el día jueves explicando más detalles de cómo va a funcionar. POR LO QUE NO ACLARAREMOS DUDAS DE ESTE TEMA POR NINGÚN OTRO MEDIO ANTES.

✨[Aquí pueden ver todos los detalles para continuar a Backend](https://fernandaochoa.github.io/innovatemplate/project/missionbackendjava) ✨

**Comment by esthersl** [comment]
Lo siento, me perdí la información. podrían decirme.
gracias

**Reply by Fernando-V** [reply]
Hoy (1/4/22) a las 7pm Mx centro será la transmisión ya que ayer no pudieron y lo anunciaron en el grupo
![image](https://user-images.githubusercontent.com/99154039/161213090-aa89dd95-2cfc-4a93-a35f-d81f4d046791.png)
